packages/libs/react-ui/src/components/Dialog/Dialog.css.ts

```diff
@@ -0,0 +1,53 @@
+import { sprinkles } from '@theme/sprinkles.css';
```

Comment:
is it correct that we can have 2 modals popping up on the same screen?
![image](https://github.com/kadena-community/kadena.js/assets/4015521/d4eb2852-c9b0-40d6-bf8f-a3a95d870aa7)

Reply:
i dont think this should be possible imho

Reply:
Hmm So I assume you were only able to achieve this by putting a Dialog inside of another Dialog since focus is shifted to the dialog when one is open. Is that correct?

Reply:
We will assume users can ensure this doesn't happen since this can only be achieved either by the same trigger opening to modals or nested modals.